### ISSUE TITLE
Any plans to support FT_Library_SetLcdGeometry?

### ISSUE THREAD

**Issue by byungil-oh** [snowie2000/mactype#792]
It looks like FreeType supports different subpixel geometry, such as PenTile arrangement, by FT_Library_SetLcdGeometry (https://www.freetype.org/freetype2/docs/reference/ft2-lcd_rendering.html)

Have you thought about supporting this feature?

**Comment by snowie2000** [comment]
Okey-dokey. Will pay more attention to it.

Thank you!

**Comment by Crayolable** [comment]
Any update for this would be sincerely appreciated. Right now, you can't adjust the subpixel geometry concisely enough for monitors like the new AW3423DW. 
For comparison, running the "ftview" command on Linux yields impressive results.

**Comment by snowie2000** [comment]
https://freetype.org/freetype2/docs/reference/ft2-lcd_rendering.html#ft_library_setlcdgeometry

Because this new tech is incompatible with our time-proven ClearType style subpixel rendering, it's not possible to introduce it without some modifications. Anyways, I'll give it a shot.

**Comment by snowie2000** [comment]
There is a test build out there just now with LCDGeometry support.
Set AntiAliasMode to 6 to enable experimental pentile support.

Looking forward to your feedback.

**Comment by Crayolable** [comment]
I've made a copy of the default profile (renamed to "Default_fix.ini") and added the following to it, but there seems to be no difference when toggling between the two:

```
[General]
PixelLayout=-21,-16,0,16,21,-16
```
Verified this in Windows Explorer, Firefox, Discord & Notepad. Quitting and then reopening any of these applications, or even restarting the system between profile changes has no effect, something I verified by taking multiple photos of the same text after each troubleshooting step.

Is there anything in particular you'd like me to try?

**Comment by snowie2000** [comment]
Could you please upload your profile so that I can try it ?

**Comment by Crayolable** [comment]
Absolutely, thank you! [Default_fix.zip](https://github.com/snowie2000/mactype/files/9150247/Default_fix.zip)

**Comment by snowie2000** [comment]
![Untitled](https://user-images.githubusercontent.com/11767189/179998292-90e63e16-b70a-41cf-b92e-9196aef87913.png)
Compare side by side, they do have a significant difference.

If you're on a hidpi display, you need to save the image to disk and view it in a image viewer. Browsers will automatically zoom it so that everything is in blur.

**Comment by Crayolable** [comment]
I'll have to try it again later and see where I went wrong then: probably just user error of some sort. 
Thank you again for your work on MacType!

**Comment by snowie2000** [comment]
You can test against your profile with our new visual tuner. It's super handy and easy to use. 

It's just only works on 32bit apps for now.
